connector.yaml

```diff
@@ -2,7 +2,39 @@ version: "1.0"
 specification:
   name: dynamodb
   summary: A DynamoDB source plugin for Conduit
-  description: A DynamoDB source plugin for Conduit, it scans the table at the beginning taking a snapshot, then starts listening to CDC events using DynamoDB streams.
+  description: |
+    ## How to build?
+
+    Run `make build` to build the connector.
+
+    ## Testing
+
+    Run `make test-integration` to run all the integration tests. Tests require Docker to be installed and running.
+    The command will handle starting and stopping docker containers for you.
+
+    If you want to run the integration tests against your AWS DynamoDB instead of docker, You must set the environment
+    variables (`AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_REGION`) before running `make test`, 
+    or before running the tests manually.
```

Comment:
This should not be added to `connector.yaml`, this file is meant to contain informations specifically for our end-users. They don't care how to build and test the connector 😉

If you want to include this, add it directly into `README.md`, outside of `readmegen` tags.